StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -15,11 +26,16 @@ protocol EventWithPayload: Event {
 }
 
 /// A protocol for user `Event`s.
-protocol UserEvent: Event {
+protocol EventWithUserPayload: EventWithPayload {
     var userId: UserId { get }
 }
 
+/// A protocol for user owner `Event`s.
+protocol EventWithOwnerPayload: EventWithPayload {
+    var createdByUserId: UserId { get }
+}
+
 /// A protocol for channel `Event`s.
-protocol ChannelEvent: Event {
+protocol EventWithChannelId: Event {
```

Comment:
Why this change?

Reply:
The name says one thing, and in fact it does not mean that the channel object will be present.